idea/data-storage/src/program/program.repo.ts

```diff
@@ -49,7 +49,7 @@ export class ProgramRepo {
       { fromDate, toDate },
       offset || 0,
       limit || PAGINATION_LIMIT,
-      ['code'],
+      ['code', { table: 'meta', columns: ['id', 'program', 'owner', 'meta'] }],
```

Comment:
Are the 'id', 'program' and 'owner' columns necessary here?

Reply:
Not necessary. I have thought maybe it will need it.